gender_novels/analysis/dependency_parsing.py

```diff
@@ -1,7 +1,11 @@
 import urllib
 from nltk.parse.stanford import StanfordDependencyParser
```

Comment:
make sure to update the dependency parser to the non-deprecated one:
`DeprecationWarning: The StanfordDependencyParser will be deprecated. Please use nltk.parse.corenlp.StanforCoreNLPDependencyParser instead`

Reply:
I looked into this a while back and ended up getting it to work, but it requires setting up a server  locally in order to communicate with the parser via REST API. It seemed too complicated so I stuck with the deprecated version.